### PR TITLE
feat: add modquery pass-through command

### PR DESCRIPTION
**Pull request by gregmagolan**

### CODE REVIEW

**Review by alexeagle — APPROVED** [review]
wow what a crazy one. We get to learn the corners of the Bazel CLI for sure

**Comment by gregmagolan** [comment]
> wow what a crazy one. We get to learn the corners of the Bazel CLI for sure

we'll probably need to gate on the underlying bazel version as well for commands like this; will leave that for a follow-up

File: pkg/aspect/modquery/modquery.go [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+package modquery
+
+import (
+	"context"
+
+	"aspect.build/cli/pkg/bazel"
+	"aspect.build/cli/pkg/ioutils"
+	"github.com/spf13/cobra"
+
```
Comment (cgrindel) [inline pkg/aspect/modquery/modquery.go:25]: nit: Blank line

**Review by cgrindel — APPROVED** [review]

File: pkg/aspect/modquery/modquery.go (hunk 1 above)
Comment (gregmagolan) [reply pkg/aspect/modquery/modquery.go:23]: 👍

**Pull request merged**